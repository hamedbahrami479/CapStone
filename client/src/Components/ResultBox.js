import React, { useState } from 'react';


const ResultBox = () => {
    return(
<input className="resultBox" type="text" id="handling" name="result"></input>
)};

export default ResultBox;