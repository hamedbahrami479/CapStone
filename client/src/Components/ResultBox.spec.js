import React from 'react';
import { render, screen } from '@testing-library/react';
import ResultBox from './ResultBox';

describe('Test for DropDown Component', () => {
  it('snapshot DropDown component', () => {
    const { asFragment } = render(<ResultBox />);
    expect(asFragment(ResultBox)).toMatchSnapshot();
  });
});
