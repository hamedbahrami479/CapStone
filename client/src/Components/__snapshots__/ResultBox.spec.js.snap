// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for DropDown Component snapshot DropDown component 1`] = `
<DocumentFragment>
  <input
    class="resultBox"
    id="handling"
    name="result"
    type="text"
  />
</DocumentFragment>
`;
